examples/run_language_modeling.py

```diff
@@ -748,6 +829,9 @@ def main():
 
         global_step, tr_loss = train(args, train_dataset, model, tokenizer)
         logger.info(" global_step = %s, average loss = %s", global_step, tr_loss)
+        if hasattr(train_dataset, "close"):
+            # For lazy loaders, clean up file handles.
+            train_dataset.close()
```

Comment:
Same question as above: where is an unclosed file handle?

Reply:
Legacy of how I was doing it before switching to linecache. Not required anymore.